packages/roosterjs-content-model/lib/modelApi/common/normalizeContentModel.ts

```diff
@@ -48,14 +48,17 @@ function normalizeParagraph(block: ContentModelParagraph) {
 
         if (segments.length == 1 && segments[0].segmentType == 'SelectionMarker') {
             segments.push(createBr(segments[0].format));
-        } else if (
-            segments.length > 1 &&
-            segments[segments.length - 1].segmentType == 'Br' &&
-            segments.some(
-                segment => segment.segmentType != 'SelectionMarker' && segment.segmentType != 'Br'
-            )
-        ) {
-            segments.pop();
+        } else if (segments.length > 1 && segments[segments.length - 1].segmentType == 'Br') {
+            const noMarkerSegments = segments.filter(x => x.segmentType != 'SelectionMarker');
```

Comment:
Is image a segment that we need br after?

Reply:
Do you mean something like:
```html
<div><img><br></div>
```
In that case, no. You can try to remove the BR and you will get the same rendering result.

Reply:
Should we check the segmentType to not add the br to this kind of element or this would make a difference?

Reply:
We don't add BR here. Do you mean the code at line 50? It will only add BR when there is only one segment and it is SelectionMarker. So if there is Image, it will not add BR.

Reply:
got it.